lib/mayaUsd/fileio/jobs/jobArgs.cpp

```diff
@@ -707,6 +707,7 @@ UsdMayaJobExportArgs::UsdMayaJobExportArgs(
     , dagPaths(dagPaths)
     , fullObjectList(fullList)
     , timeSamples(timeSamples)
+    , exportRoots(extractVector<std::string>(userArgs, UsdMayaJobExportArgsTokens->exportRoots))
     , rootMapFunction(PcpMapFunction::Create(
```

Comment:
Previously the export roots that were given were not kept explicitly but indirectly in the rootMapFunction and were thus not accessible for code to check later on, so they are now also kept around.